content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -1,72 +1,92 @@
 
+Generate leads
+==============
 
-Start generating leads
-==========================
-You will now have a new button **Generate Leads** available in your pipeline.
-You are also able to create lead mining requests from the
-:menuselection:`Configuration --> Lead Mining Requests` and through
-:menuselection:`Leads --> Leads` where you have the **Generate Leads** button.
+After the :guilabel:`Lead Mining` setting is activated, a new button called :guilabel:`Generate
+Leads` is available to use in the :guilabel:`CRM` pipeline. Lead mining requests are also available
+through :menuselection:`CRM --> Configuration --> Lead Mining Requests`, or through
+:menuselection:`CRM --> Leads --> Leads` where the :guilabel:`Generate Leads` button is also
+available.
 
-.. image:: lead_mining/LM2.png
+.. image:: lead_mining/generate-leads-button.png
    :align: center
+   :alt: The Generate Leads button to use the lead mining feature.
 
+Click on the :guilabel:`Generate Leads` button, and a window will appear offering a variety of
+criteria by which to generate leads.
 
-From there, click on the **Generate Leads** button, a window where you will be able to pick your criteria will pop up.
+.. image:: lead_mining/generate-leads-popup.png
+   :align: center
+   :alt: The pop-up window with the selection criteria in order to generate leads in Odoo.
 
+Choose to generate leads for :guilabel:`Companies` to get company information only, or choose
+:guilabel:`Companies and their Contacts` to get company information as well as individual employee
+contact information. When targeting :guilabel:`Companies and their Contacts`, there is an option to
+filter contacts based on :guilabel:`Role` or :guilabel:`Seniority`.
 
-.. image:: lead_mining/LM5.png
-   :align: center
+Additional filtering options include:
 
+- :guilabel:`Size`: filter leads based on the number of employees at the company
+- :guilabel:`Countries`: filter leads based on the country (or countries) they are located in
+- :guilabel:`States`: further filter leads based on the state they are located in, if applicable
+- :guilabel:`Industries`: filter leads based on the specific industry they work in
+- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to
+- :guilabel:`Salesperson`: choose which person(s) on the Sales Team the leads will be assigned to
+- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found
 
-When choosing to target Companied and their contacts you can choose the contacts you are getting based on Role or Seniority. 
-When getting contact information make sure to be aware of the latest EU regulation, get more information about General Data Protection Regulation on
-`Odoo GDPR <http://odoo.com/gdpr>`__, 
+.. note::
+   Make sure to be aware of the latest EU regulations when receiving contact information. Get more
+   information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`__.
```

Comment:
Only [one underscore is needed ](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#external-hyperlinks)for the link.
```suggestion
   information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`_.
```